recipes/cuda-pathfinder/recipe.yaml

```diff
@@ -0,0 +1,54 @@
+context:
+  name: cuda-pathfinder
+  version: "1.0.0"
+  # no-arch packages can't use selectors, so we avoid using the "match(python, "<3.9")" selector this way
+  python_min: "3.9"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  # TODO: switch to use the stable release url in the next release
+  url: https://github.com/nvidia/cuda-python/archive/${{ name }}-v${{ version }}.tar.gz
```

Comment:
Why can't we use the release url?

```https://github.com/NVIDIA/cuda-python/archive/refs/tags/${{ name }}-v${{ version }}.tar.gz```

Reply:
Because it's what recommended in the example v1 recipe:
https://github.com/conda-forge/staged-recipes/blob/bcdb5eacff631bb042d0d347d2ea37a58aad223c/recipes/example-v1/recipe.yaml#L24-L28
I think both archive urls are the same thing, and both have the same unstable hash issue so we need to change it next time we make a release (we could not this time because the release workflow was not used, so no artifact was attached to the release).